app/presenters/publishing_api/world_location_news_article_presenter.rb

```diff
@@ -60,6 +60,16 @@ def image_available?
       item.images.any?
     end
 
+    def prepend_slash(path)
+      path.start_with?("/") ? path : path.prepend("/")
+    end
+
+    def image_path
+      image_path = presented_world_location_news_article.lead_image_path
+
+      Whitehall.public_asset_host + prepend_slash(image_path)
```

Comment:
That would be misappropriating `File`, lucky though `URI` has it's own `join` which does exactly what is needed!

```rb
URI.join(Whitehall.public_asset_host, "/", "foo.png")
# => #<URI::HTTP http://assets-origin.dev.gov.uk/foo.png>
URI.join(Whitehall.public_asset_host, "/", "/foo.png")
# => #<URI::HTTP http://assets-origin.dev.gov.uk/foo.png>
```